R/db2.R

```diff
@@ -123,21 +123,21 @@ report_db2_init <- function(con, config, must_create = FALSE, validate = TRUE) {
 }
 
 
-report_db2_init_create <- function(con, config) {
-  if (config$destination$driver[[1]] == "RSQLite") {
-    dialect <- "sqlite"
-  } else {
-    dialect <- "postgres"
-  }
+report_db2_init_create <- function(con, config, dialect) {
```

Comment:
added a `dialect` argument to make the postgres branch a little more testable - we should still add a full postgres test though on travis https://vimc.myjetbrains.com/youtrack/issue/VIMC-2417